pallets/ema-oracle/src/lib.rs

```diff
@@ -142,6 +145,13 @@ pub mod pallet {
 		/// Whitelist determining what oracles are tracked by the pallet.
 		type OracleWhitelist: Contains<(Source, AssetId, AssetId)>;
 
+		/// Asset id converter based on xcm locations
+		type CurrencyIdConvert: sp_runtime::traits::Convert<Location, Option<AssetId>>;
+
+		/// Maximum allowed percentage difference for bifrost oracle price update
+		#[pallet::constant]
+		type MaxAllowedPriceDifference: Get<(u32, u32)>;
```

Comment:
why not Permill? wouldnt it be more clear?

Reply:
yes, percent or permill pls